Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "MIT OR Apache-2.0"
 name = "prima-tracing"
 readme = "README.md"
 repository = "https://github.com/primait/prima_tracing.rs"
-version = "0.9.0-rc.0"
+version = "0.9.0-rc.1"
```

Comment:
```suggestion
version = "0.9.1"
```

Reply:
For now I'll keep this an RC, once it's stable I'll release a 0.9.0

Reply:
Ah sorry i was sure that the 0.9.0 already existed. Fine then!